### PR TITLE
Partial time series fill

### PR DESCRIPTION
**Pull request by JensGM**
Bazefetcher output now lets you write into existing time series, but while keeping existing indices.

### CODE REVIEW

File: camille/output/bazefetcher.py [hunk 1]
```diff
@@ -5,46 +5,6 @@
 from camille.source.bazefetcher import _tidy_frame
 
 
-def _to_midnight_utc(timestamp):
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/output/bazefetcher.py]: Hmmmmmmmmmmm

[the remark this answers is not in the document]
Comment (JensGM) [reply camille/output/bazefetcher.py]: I checked, yes, I'm sure :D

File: camille/output/bazefetcher.py [hunk 2]
```diff
@@ -66,22 +26,29 @@ def _generate_tag_location(
     return path
 
 
-def _merge(ts, into, overwrite=False):
+def _merge(ts, into, overwrite=False, fill=False):
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/output/bazefetcher.py]: Not here, but farther down, yes

File: camille/output/bazefetcher.py [hunk 3]
```diff
@@ -192,18 +160,17 @@ def __call__(self,
...
-        for s, e in _daterange( start, end ):
-            tag_path = _generate_tag_location( self.root,
-                                               tag,
-                                               s,
-                                               e,
-                                               full_path=True,
-                                               suffix='.json.gz' )
-
-            ts = series[s:e-eps]
-            if ts.empty: continue
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/output/bazefetcher.py]: With this change, empty days never get in the loop

File: camille/output/bazefetcher.py [hunk 4]
```diff
@@ -192,18 +160,17 @@ def __call__(self,
...
+        for d, view in series.groupby(series.index.date):
```
[the remark this answers is not in the document]
Comment (JensGM) [reply camille/output/bazefetcher.py]: :D

File: camille/output/bazefetcher.py [hunk 5]
```diff
@@ -174,6 +143,10 @@ def __call__(self,
             to be written, is deleted.
             False - raise a ValueError on overwrite attempt.
             Default is False
+        fill : bool, optional
```
Comment (achaikou) [inline camille/output/bazefetcher.py]: How does it play with overwrite? 
ovewrite=false, fill = true - only insert new data
overwrite=true - totally overwrite the data, regardless of fill value

**Pull request merged**